Corona-Warn-App/src/test/java/de/rki/coronawarnapp/covidcertificate/signature/core/DscSignatureValidatorTest.kt

```diff
@@ -44,6 +46,9 @@ class DscSignatureValidatorTest : BaseTest() {
         every { dscRepository.dscData } returns flowOf(dscData)
         every { securityProvider.setup() } just Runs
 
+        // Add RSA algorithm in unit tests context.
+        Security.addProvider(BouncyCastleProvider())
```

Comment:
Does this need to removed again?

Reply:
No, it will stay. SecurityProvider.setup() should NOT be mocked ,but the crypto but `org.conscrypt.*` is not available for unit testing. we could use an openKJDK implementation ,but it is not really worth it for one unit test, more information here. https://github.com/google/conscrypt/issues/649

Reply:
For the actual App `SecurityProvider` will be doing that ,but it is a problem for the real app